Analytics/Integrations/Flurry/SEGFlurryIntegration.m

```diff
@@ -62,6 +62,16 @@ - (void)identify:(NSString *)userId traits:(NSDictionary *)traits options:(NSDic
   if (age) {
     [Flurry setAge:[age intValue]];
   }
+  
+  NSDictionary *location = [traits objectForKey:@"location"];
+  
+  if (location) {
+    float latitude = [[location objectForKey:@"latitude"] floatValue];
```

Comment:
So what happens if `[location objectForKey:@"latitude"] ` is `nil`? `latitude` simply gets set to 0?

The way I was reading this (as in Java) was `[nil floatValue]` would throw a NullPointerException.